File: 2543.py
while True:
    try:
        n, id = input().split(' ')
        soma = 0
        for i in range(0, int(n)):
            ID, escolha = input().split(' ')
            if id == ID and escolha == '0':
                soma+=1
        print(soma)
    except EOFError:
        break

File: 2543.cpp
#include <iostream>
using namespace std;

int main(){
    int numero_casos, id, chave, escolha, soma;
    while(cin >> numero_casos){
        soma = 0;
        cin >> id;
        for(int i = 0; i < numero_casos; i++){
            cin >> chave >> escolha;
            if(id == chave && escolha == 0)
                soma++;
        }
        cout << soma << endl;
    }
    return 0;        
}
